.semaphore/semaphore.yml

```diff
@@ -43,6 +43,10 @@ global_job_config:
       - echo $SEMAPHORE_WORKFLOW_ID
       - echo $SEMAPHORE_GIT_REPO_SLUG
       - checkout
+      - |
+        if [[ "${SEMAPHORE_ORGANIZATION_URL}" != *".semaphoreci.com" ]]; then
+          . vault-setup
```

Comment:
You could move this call to the block below to removal a conditional, but this is fine